Project/src/MakeCall/CallCard.js

```diff
@@ -216,6 +218,18 @@ export default class CallCard extends React.Component {
                 });
             });
 
+            const mediaCollector = this.call.feature(Features.MediaStats).createCollector();
+            mediaCollector.on('sampleReported', (data) => {
```

Comment:
please look into adding off or dispose as well to avoid memory leaks

Reply:
It will not leak because the MediaStatsCollector is designed to dispose automatically when call ends (and also release the listener)